templates/default/sysvinit.service.erb

```diff
@@ -64,7 +64,7 @@ _start() {
     daemon \
          --pidfile=$pidfile \
          --user=$user \
-         " { $exec <%= @daemon_options %> &>> $logfile & } ; echo \$! >| $pidfile "
+         " { $exec <%= @daemon_options %> > $logfile 2>&1 & } ; echo \$! >| $pidfile "
```

Comment:
What is your reason for doing this? This means that the log file will be overwritten.